admin/services/service-message.service.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const redisCacheService = require('../services/redis-cache.service')
+
+const serviceMessageService = {}
+const redisKey = 'serviceMessage'
+
+/**
+ * Get the service message
+ * @returns {object}
+ */
+serviceMessageService.getMessage = async () => {
+  const redisResult = await redisCacheService.get(redisKey)
+  return JSON.parse(redisResult)
```

Comment:
JSON.parse can throw.  Is that fine?

Reply:
its wrapped in a try/catch block from controller level. question is do we need additional error handling for this? 🤔

Reply:
@jon-shipley outdated block